lib/knapsack/presenter.rb

```diff
@@ -64,8 +69,8 @@ def time_offset_warning
         str << "\nhttps://docs.knapsackpro.com/ruby/knapsack#faq"
         str << "\n=================================================\n"
         str << %{See how to split tests in a dynamic way using Queue Mode to ensure all parallel nodes
-finish work at a similar time even when your CI or random test execution is a bottleneck:
-https://youtu.be/hUEB1XDKEFY
+finish work at a similar time even when your CI server or a random tests execution time is a bottleneck:
```

Comment:
This might be a little unclear to some imho. Maybe we can simplify this to sth plain like:

`Read up on the benefits of a dynamic test split with Knapsack's Queue Mode:`

What do you think?

Reply:
good point. The article explains it.